index.html

```diff
@@ -17,202 +18,174 @@
-          <p class="about-title">Learn CSS</p>
-          <p class="about-content">Learn about CSS from scratch, how to put styles to web pages, learn about frameworks and more!!!</p>
+        <div class="column ">
+          <i><img class="tech-icons" src="./images/icons8-css3-48.png" alt=""></i>
+          <p class="about-title"><strong>Learn CSS </strong> </p>
+          <p class="about-content">Learn about CSS from scratch, how to put styles to web pages, learn about frameworks
+            and more!!!</p>
         </div>
 
-        <div class="column col-12 col-md-4">
-          <i><img src="./images/icons8-javascript-48.png" alt=""></i>
-          <p class="about-title">Learn Javascript</p>
-          <p class="about-content">Learn about Javascript from scratch, the interaction of the web pages, learn about some different scripts and more!!!</p>
+        <div class="column  ">
+          <i><img class="tech-icons" src="./images/icons8-javascript-48.png" alt=""></i>
+          <p class="about-title"><strong>Learn Javascript</strong> </p>
+          <p class="about-content">Learn about Javascript from scratch, the interaction of the web pages, learn about
+            some different scripts and more!!!</p>
         </div>
-        <div class="badges-contenedor">
-          <span class="badge text-bg-danger">Join to our 2023 program, right now!!!</span>
-        </div>       
-      </div>
-
-    </div>
-  </section>
 
-  <section class="featured-speakers">
-    <div class="container">
-      <p class="text-center"><strong>Featured Speakers </strong>  </p>
-      <hr>
-  
-    </div>
+      </div>
+      <div class="hidding-p">
+        <a href="">
+          <p class="text-center section-p"><strong>SEE THE HOLE PROGRAM</strong> </p>
+        </a>
 
-    <div class="container text-enter about-main-programs">
 
-      <div class="column2 ">
-        <i><img class="speaker" src="./images/speaker1.png" alt=""></i>
-        <div class="speaker-content">
-          <h2>Yochai Benkler</h2>
-          <p class="about-title2">Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School</p>
-          <hr>
-          <p class="about-content">Benkler studies commons-bades peer production, and published his semnial book The Wealth of Networks in 2006.</p>
-        </div>
-        
       </div>
 
-      <div class="column2 ">
-        <i><img class="speaker" src="./images/speaker2.png" alt=""></i>
-        <div class="speaker-content">
-          <h2>SohYeong Noh</h2>
-          <p class="about-title2">Director of Art Centre Nabi and a board member of CC </p>
-          <hr>
-          <p class="about-content">Benkler studies commons-bades peer production, and published his semnial book The Wealth of Networks in 2006.</p>
-        </div>
-        
+      <div class="badges-contenedor text-center">
+        <span class="badge text-bg-danger">Join to our 2023 program, right now!!!</span>
       </div>
```

Comment:
Kindly adjust your button styling to match the original design

Your design
![image](https://user-images.githubusercontent.com/78251927/233224258-58c0d3ab-1357-4c9b-9def-5607a8a7cd8b.png)

Template
![image](https://user-images.githubusercontent.com/78251927/233224289-63289e68-679b-4cde-a73a-4dd8c061f822.png)

Reply:
![image](https://user-images.githubusercontent.com/111823606/233439463-5192212c-e489-4360-a38a-565c6ae300d3.png)
Got it!!!